powertools-parameters/powertools-parameters-dynamodb/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>software.amazon.lambda</groupId>
+        <artifactId>powertools-parent</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>powertools-parameters-dynamodb</artifactId>
+    <name>Powertools for AWS Lambda (Java) library Parameters - DynamoDB</name>
+    <description>DynamoDB implementation for the Parameters module</description>
+
+    <issueManagement>
+        <system>GitHub Issues</system>
+        <url>https://github.com/aws-powertools/powertools-lambda-java/issues</url>
+    </issueManagement>
+    <scm>
+        <url>https://github.com/aws-powertools/powertools-lambda-java.git</url>
+    </scm>
+    <developers>
+        <developer>
+            <name>Powertools for AWS Lambda team</name>
+            <organization>Amazon Web Services</organization>
+            <organizationUrl>https://aws.amazon.com/</organizationUrl>
+        </developer>
+    </developers>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://aws.oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+    </distributionManagement>
```

Comment:
you can clean this I think ;) and same for other modules

Reply:
Where have I heard this before